tests/test_serving.py

```diff
@@ -129,12 +129,13 @@ def test_windows_get_args_for_reloading(monkeypatch, tmp_path):
 @pytest.mark.filterwarnings("ignore::pytest.PytestUnraisableExceptionWarning")
 @pytest.mark.parametrize("find", [_find_stat_paths, _find_watchdog_paths])
 def test_exclude_patterns(find):
-    # Imported paths under sys.prefix will be included by default.
+    # Get a path to exclude
```

Comment:
This comment needs to be more descriptive, such as "pick a path that contains files to exclude".

Reply:
will do